trade-mapping-to-ipaffs/src/main/java/uk/gov/defra/stw/mapping/toipaffs/cheda/purpose/ChedaReEntryPurposeMapper.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.defra.stw.mapping.toipaffs.cheda.purpose;
+
+import static uk.gov.defra.tracesx.notificationschema.representation.enumeration.ForImportOrAdmissionEnum.HORSES_RE_ENTRY;
+import static uk.gov.defra.tracesx.notificationschema.representation.enumeration.PurposeGroupEnum.RE_IMPORT;
+
+import org.springframework.stereotype.Component;
+import uk.gov.defra.stw.mapping.dto.SpsCertificate;
+import uk.gov.defra.stw.mapping.toipaffs.Mapper;
+import uk.gov.defra.tracesx.notificationschema.representation.Purpose;
+
+@Component
+public class ChedaReEntryPurposeMapper implements Mapper<SpsCertificate, Purpose> {
+
+  @Override
+  public Purpose map(SpsCertificate spsCertificate) {
```

Comment:
Not really sure how we'd do that because `ChedaPurposeMapper` is using the factory pattern to get the correct mapper to use based on the `PURPOSE` value. This is just a mapper that returns static values. It has to have SpsCertificate in the argument so it conforms to the type `Mapper<SpsCertificate, Purpose>` in `private final Map<String, Mapper<SpsCertificate, Purpose>> purposeMappers;`

Reply:
I could see that you'd done an implementation of the factory pattern.  Just seems a bit excessive, but I can see what you mean now with it needing to conform to the Mapper interface.  Resolving